aspnetcore/security/authentication/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Overview of ASP.NET Core Authentication
+author: mjrousos
+description: Learn about authentication in ASP.NET Core.
+ms.author: riande
+ms.custom: mvc
+ms.date: 11/22/2019
+uid: security/authentication/index
+---
+# Overview of ASP.NET Core authentication
+
+By [Mike Rousos](https://github.com/mjrousos)
+
+Authentication is the process of determining a user's identity. [Authorization](xref:security/authorization/introduction) is the process of determining whether a user has access to a resource. In ASP.NET Core, authentication is handled by authentication [middleware](xref:fundamentals/middleware/index). The authentication middleware uses registered authentication handlers to complete authentication-related actions. Examples of authentication-related actions include:
+
+* Authenticating a user.
+* Responding when an unauthenticated user tries to access a restricted resource.
+
+The registered authentication handlers and their configuration options, are called "schemes".
+
+Authentication schemes are specified by registering authentication services in `Startup.ConfigureServices`. This is done by:
+
+* Calling [AuthenticationBuilder.AddScheme](xref:Microsoft.AspNetCore.Authentication.AuthenticationBuilder.AddScheme*)
+* Calling a scheme-specific extension methods that call `AddScheme` automatically with appropriate settings. This approach is more common than calling `AuthenticationBuilder.AddScheme`.
+
+For example, the following code registers authentication services and handlers for cookie and JWT bearer authentication schemes:
+
+```csharp
+services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+    .AddJwtBearer(options => Configuration.Bind("JwtSettings", options))
+    .AddCookie(options => Configuration.Bind("CookieSettings", options));
+```
+
+The `AddAuthentication` parameter `JwtBearerDefaults.AuthenticationScheme` is the name of the scheme to use by default if a specific one isn't requested. In the preceding code, JWT bearer authentication is used by default.
```

Comment:
Might want to expand on this. Perhaps add cookie with a specific scheme name to show the contrast?

Reply:
Good idea. I'll do that.